backend/infrahub/core/migrations/schema/node_attribute_remove.py

```diff
@@ -22,8 +23,20 @@ async def query_init(self, db: InfrahubDatabase, **kwargs: dict[str, Any]) -> No
         branch_filter, branch_params = self.branch.get_query_filter_path(at=self.at.to_string())
         self.params.update(branch_params)
 
+        attr_name = self.migration.schema_path.field_name
+        kinds_to_ignore = []
+        if isinstance(self.migration.new_node_schema, GenericSchema):
+            for inheriting_schema_kind in self.migration.new_node_schema.used_by:
+                node_schema = db.schema.get_node_schema(
+                    name=inheriting_schema_kind, branch=self.branch, duplicate=False
+                )
+                attr_schema = node_schema.get_attribute_or_none(name=attr_name) if attr_name else None
```

Comment:
When `attr_name` could be empty here? Should we early return as the query would return have no effect?
Otherwise, I think `if attr_name` check could be performed line 28 so we don't loop if it's `None`.